### ISSUE TITLE
Stuck in inconsistent connection state

### ISSUE THREAD

**Issue by haarp** [jcjordyn130/simpleirc#4]
Under many circumstances it's possible to get the app stuck in an inconsistent state, meaning it thinks its connected when it isn't, it thinks it's disconnected when it's working, it thinks its connected but still throws up a "do you want to reconnect" query when tapping a network, it pretends to be connected when it isn't, etc.

It's often impossible to escape this state without quitting the app entirely.

also see https://github.com/indrora/Atomic/issues/71

**Comment by jcjordyn130** [comment]
Yes I've been aware of this issue for a while now, and I'm working on a fix on it. But you know, life gets in the way.

**Comment by jcjordyn130** [comment]
Also as of now I'm the only dev.
